RetailCoder.VBE/Refactorings/ExtractMethod/ExtractMethodRefactoring.cs

```diff
@@ -17,11 +18,13 @@ namespace Rubberduck.Refactorings.ExtractMethod
     public class ExtractMethodRefactoring : IRefactoring
     {
         private readonly VBE _vbe;
+        private readonly RubberduckParserState _state;
         private readonly IRefactoringPresenterFactory<IExtractMethodPresenter> _factory;
 
-        public ExtractMethodRefactoring(VBE vbe, IRefactoringPresenterFactory<IExtractMethodPresenter> factory)
+        public ExtractMethodRefactoring(VBE vbe, RubberduckParserState state, IRefactoringPresenterFactory<IExtractMethodPresenter> factory)
```

Comment:
won't this conflict with @grleachman's changes? he's been working specifically on that refactoring for two weeks...